kernel_prerm.d_dkms

```diff
@@ -7,7 +7,15 @@ if command -v dkms > /dev/null; then
 	dkms status -k "$inst_kern" 2>/dev/null | while IFS=",:/ " read -r name vers _ arch status; do
 		[ "$status" = "installed" ] || continue
 		echo "dkms: removing: $name $vers ($inst_kern) ($arch)" >&2
-		dkms remove -m "$name" -v "$vers" -k "$inst_kern" -a "$arch"
+		# Compromise on using 'unbuild' to remove the module when a
```

Comment:
> Since I'm on the forgetful side, please mention when this triggers. Say "This script triggers when the kernel/ kernel-headers/both kernel and kernel-headers package(s) is/are removed".

OK, done.